services/auth-basic/README.md

```diff
@@ -0,0 +1,25 @@
+# Auth-Basic Service
+
+The `auth-basic` service is responsible for validating authentication of incoming requests. To do so it will forward the call to an `auth manager`
+(see `Auth Managers` section). Almost every request to the server will involve the `auth-basic` service as access tokens need to be verified.
+
+## Auth Managers
+
+Since the `auth-basic` service does not do any validation itself, it needs to be configured with an auth manager. One can use the `AUTH_BASIC_AUTH_PROVIDER` envvar to configure this.
+
+### LDAP Auth Manager
+
+Setting `AUTH_BASIC_AUTH_PROVIDER` to `"ldap"` will configure the `auth-basic` service to use ldap as auth manager. This is the recommended option for productive and testing deployments. 
+More details on how to configure ldap with ocis can be found in the admin docs.
```

Comment:
```suggestion
Setting `AUTH_BASIC_AUTH_PROVIDER` to `"ldap"` will configure the `auth-basic` service to use ldap as auth manager. This is the recommended option for productive and testing deployments. More details on how to configure LDAP with ocis can be found in the admin docs.
```
aha, interesting - where? (nice try 😅 )

Reply:
Also "productive" => "production".

In English we would say "the software is running in a production environment". I suppose Europeans are writing "productive environment" because of some different way the the corresponding words are using in European languages?

Reply:
changed, thanks for clarifying